spec/index.bs

```diff
@@ -1414,7 +1422,7 @@ Because of this, and to improve portability across devices and implementations,
 applications should generally request the "worst" limits that work for their content
 (ideally, the default values).
 
-A <dfn dfn>supported limits</dfn> object has a value for every limit defined by WebGPU:
+A <dfn dfn>supported limits</dfn> object has an entry for every limit defined by WebGPU:
```

Comment:
It's clear enough what's intended, but it does worry me a bit that we don't say what type of value a supported limits object is, in terms of the [infra types](https://infra.spec.whatwg.org/), say. Isn't it a dictionary?

Reply:
It probably would be good to base this on infra types. I've never realized but it defines something called [=struct=] which I've never looked at (because it defines almost nothing about it). There are actually several places in the spec where we would want to define types as structs or tuples. I have filed an issue about this: #4800

(For some history, I think [=supported limits=] and {{GPUSupportedLimits}} used to be the same object, so some of the language around [=supported limits=] treats it kind of like a WebIDL `interface`.)